relayer/create_ibc_channel.go

```diff
@@ -34,15 +44,48 @@ func (r *Relayer) CreateIBCChannel(override bool, logFileOption utils.CommandOpt
 		return ConnectionChannels{}, err
 	}
 
+	// We always pass override otherwise this commands hangs if there are too many clients
+	// in the hub as it iterates all to check if this client exists
+	createClientsCmd := r.getCreateClientsCmd(true)
+	status = "Creating clients..."
+	fmt.Printf("💈 %s\n", status)
+	if err := r.WriteRelayerStatus(status); err != nil {
+		return ConnectionChannels{}, err
+	}
+	if err := utils.ExecBashCmd(createClientsCmd, logFileOption); err != nil {
+		return ConnectionChannels{}, err
+	}
+
+	// Sleep for a few seconds to make sure the clients are created
+	// otherwise the connection creation attempt fails
+	time.Sleep(10 * time.Second)
+
+	connectionID, _ := r.GetActiveConnection()
+	if connectionID == "" || override {
```

Comment:
if u set `override=true` in the createClients, this check is wrong.
if creating new client, u will must to create new connection as well